sale_order_line_clone/README.rst

```diff
@@ -0,0 +1,87 @@
+=====================
+Sale Order Line Clone
+=====================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:d249d5d8b3c312214f35c525cdbacc99f98cdec58b2c81b11e9ccfa247acc698
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-sygel--technology%2Fsy--sale--workflow-lightgray.png?logo=github
+    :target: https://github.com/sygel-technology/sy-sale-workflow/tree/15.0/sale_order_line_clone
+    :alt: sygel-technology/sy-sale-workflow
+
+|badge1| |badge2| |badge3|
+
+This module allows you the option to clone sale order lines.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+# Just install.
+
+Configuration
+=============
+
+# Go to *Settings > Users and Company > Users* and edit users how can
```

Comment:
Please, change this to #. In all the readme (add the .)

This is not properly displayed

Reply:
Changed